insights/parsers/qemu_xml.py

```diff
@@ -1,13 +1,51 @@
 """
-QemuXML file ``/etc/libvirt/qemu/*.xml``
-----------------------------------------
+QemuXML file ``/etc/libvirt/qemu/*.xml`` and ``/var/run/libvirt/qemu/*.xml``
+----------------------------------------------------------------------------
```

Comment:
This should be a line of `===` chars.  If you look at the rendered output it creates three headings at the same level instead of 1 major heading and two subheadings.